services/tenant-ui/frontend/package.json

```diff
@@ -7,8 +7,8 @@
     "dev": "vite",
     "build": "vue-tsc --noEmit && vite build",
     "preview": "vite preview",
-    "lint": "eslint --ext .ts,vue --max-warnings=0 .",
-    "lintfix": "eslint --fix --ext .ts,vue .",
+    "lint": "cross-env ESLINT_USE_FLAT_CONFIG=false eslint --ext .ts,vue --max-warnings=0 .",
+    "lintfix": "cross-env ESLINT_USE_FLAT_CONFIG=false eslint --fix --ext .ts,vue .",
```

Comment:
Temp so FE can still find it's lint file without walking up the directory to find the BE flat file. (upgrade FE to eslint 9 later)